artemis/modules/pdf_censor_scanner.py

```diff
@@ -0,0 +1,45 @@
+from enum import Enum
+from typing import Any, Dict, List
+
+import xray
+from karton.core import Task
+
+from artemis.binds import Service, TaskStatus, TaskType
+from artemis.module_base import ArtemisBase
+from artemis.task_utils import get_target_url
+
+
+class PDFScannerFindings(Enum):
+    CENSORSHIP_WEAKNESS = "censorship_weakness"
+
+
+class PDFCensorScanner(ArtemisBase):
```

Comment:
I would rather name that LeakScanner, as badly censored PDFs may be only one of types of leaked data, and all leak scans will require the same core functionality of crawling?